src/eynollah/eynollah.py

```diff
@@ -311,20 +327,15 @@ def __init__(
         if self.ocr and self.tr:
             self.model_ocr_dir = dir_models + "/model_eynollah_ocr_trocr_20250919"
         elif self.ocr and not self.tr:
-            self.model_ocr_dir = dir_models + "/model_eynollah_ocr_cnnrnn_20250904"
+            self.model_ocr_dir = dir_models + "/model_eynollah_ocr_cnnrnn_20250930"
```

Comment:
@vahidrezanezhad `model_eynollah_ocr_cnnrnn_20250930` is `model_eynollah_ocr_cnnrnn_20250904` but with a transposed `Conv1D` layer to avoid the NCHW issue (right? @bertsky)

```diff
-    out = tensorflow.keras.layers.Conv1D(max_seq, 1, data_format="channels_first" (addition_rnn)

+    addition_rnn = tf.transpose(addition_rnn, perm=[0, 2, 1])
+    out = tensorflow.keras.layers.Conv1D(max_seq, 1, data_format="channels_last")(addition_rnn)
+    out = tf.transpose(out, perm=[0, 2, 1])
```

How can we solve this in eynollah itself?

Reply:
Yes, exactly, that's the change in `build_model()`.

No, we cannot solve this at runtime, because we just `tensorflow.keras.models.load_model()` the SavedModel format directory and then call `predict()` on it. There is no way to access and manipulate the layers after that AFAIK. (Remember the model is now a compiled functional execution graph which does not have a correspondence to the lines of code.)

Reply:
It can be scripted by:
1. loading the old model via `tensorflow.keras.models.load_model`
2. saving its weights separately via `model.save_weights`
3. running the modified `build_model` function to create a new model
4. load the saved weights into that model via `model.load_weights`
5. saving the entire model under a new name via `model.save`